src/pages/case-studies.js

```diff
@@ -1,38 +1,137 @@
+import React, { useState } from 'react';
 import '../sass/styles.scss';
-import React from 'react';
 import RedLayout from '../layouts/red';
 import { graphql, useStaticQuery } from 'gatsby';
-import {Helmet} from "react-helmet";
+import { Helmet } from 'react-helmet';
 import Hero from '../components/hero.js';
+import PrimaryPageCTA from '../components/primary-page-cta';
 
 const CaseStudyPage = () => {
-  const data = useStaticQuery(query);
-  return (
-    <RedLayout>
-        <Hero
-            title='Work that makes a difference'
-            description='Our work impacts the daily lives of millions of people. See how we’ve helped agencies build resilient services at scale.'
-        />
-        <Helmet>
-            <title data-react-helmet="true">Case Study Landing page</title>
-        </Helmet>
-      <pre>{JSON.stringify(data, null, 4)}</pre>
-    </RedLayout>
-  );
-};
+  const data = useStaticQuery(graphql`
+    {
+      allStrapiCaseStudy {
+        nodes {
+          Title
+          Cover_Image {
+            url
+            alternativeText
+            caption
+          }
+          Client_Name
+          Path
+          Service_Category {
+            Category
+          }
+          Sort_Order
+        }
+      }
+    }
+  `);
+
+
 
-export const query = graphql`
-  {
-    allStrapiCaseStudy {
-      nodes {
-        id
-        Path
-        Style
-        Title
-        Client_Name
+  const categories = [
+    { id: 1, value: 'Web & CMS', name: 'Web_CMS', isChecked: false },
+    {
+      id: 2,
+      value: 'IT & Service Modernization',
+      name: 'IT_Service_Modernization',
+      isChecked: false,
+    },
+    {
+      id: 3,
+      value: 'Product & Design',
+      name: 'Product_Design',
+      isChecked: false,
+    },
+    {
+      id: 4,
+      value: 'Security & Compliance',
+      name: 'Security_Compliance',
+      isChecked: false,
+    },
+    { id: 5, value: 'Data Services', name: 'Data_Services', isChecked: false },
+    {
+      id: 6,
+      value: 'Workforce Development',
+      name: 'Workforce_Development',
+      isChecked: false,
+    },
+  ];
+
+  const [filterState, setFilterState] = useState(null);
+  // Handle onChange from Filters
+  const callback = (e) => {
+    if (e.target.name !== 'undefined') {
+      if (filterState === e.target.name) {
+        // The button was already selected.
+        setFilterState(null);
+      } else {
+        setFilterState(e.target.name);
       }
     }
+  };
+
+  let cases;
+  const caseStudies = data.allStrapiCaseStudy.nodes;
+
+  // Handle filter states
+  if (filterState) {
+        // Todo: Only handles one input filter at a time, not multiple
```

Comment:
@iris-i I can filter the data based on the input filters but only one at a time. Ive tried everything to get it working to accept multiple checks.

Reply:
@jaube05 I made some changes to make this accept multiple filters. Lmk if it works as expected.

Reply:
yes it worked thank you